README.md

```diff
@@ -84,6 +86,106 @@ Local development uses [Laravel Herd](https://herd.laravel.com/docs/1/getting-st
     openssl rand -hex 32
     ```
 
+   Add it to your `.env` file:
+
+    ```dotenv
+    CIPHERSWEET_KEY="<your key>"
+    ```
+
+6. Install Composer and NPM dependencies:
+    ```bash
+    # install composer dependencies
+    composer install
+    # To use the version of npm specified in .nvmrc.
+    # requires https://github.com/nvm-sh/nvm
+    nvm use
+    # install node dependencies
+    npm ci
+    ```
+7. Generate an application key:
+     ```bash
+     php artisan key:generate
+     ```
+8. Create a database for development and one for running tests:
+    ```bash
+    mysql -uroot -e "create database accessibilityexchange;"
+    mysql -uroot -e "create database tae-test;"
+    ```
+9. Run the required database migrations:
+     ```bash
+     php artisan migrate
+     ```
+10. Download the application fonts:
+    ```bash
+    php artisan google-fonts:fetch
+    ```
+11. Tell Herd to serve the application:
+      ```bash
+      herd link
+      ```
+12. Install [Mailpit](https://github.com/axllent/mailpit) so that you can access transactional email from the platform:
+    ```bash
+    brew install mailpit
+    brew services start mailpit
+    ```
+    Then, make sure that your `.env` file contains the following values:
+    ```dotenv
+    MAIL_MAILER=smtp
+    MAIL_HOST=127.0.0.1
+    MAIL_PORT=1025
+    ```
+    You will now be able to access mail that the platform sends by visiting http://127.0.0.1:8025 or http://localhost:8025. For more information and additional configuration options, [read the Mailpit documentation](https://github.com/axllent/mailpit).
+
+For comprehensive instructions, consult the [Laravel documentation](https://laravel.com/docs/10.x). Here's an overview
+of how some key tasks can be carried out using Valet:
+- [Composer](https://getcomposer.org) commands may be executed by using `composer <command>`.  
+- [NVM](https://github.com/nvm-sh/nvm) commands may be executed by using `nvm <command>`.  
+- [NPM](https://docs.npmjs.com/cli) commands may be executed by using `npm <command>`.  
+- [Artisan](https://laravel.com/docs/10.x/artisan) commands may be executed by using `php artisan <command>`.  
+
+
+### Local development setup using Laravel Valet
```

Comment:
The Valet docs aren't needed at all and can be removed.

Reply:
Done